cmd/openshift-install/create.go

```diff
@@ -288,11 +286,9 @@ func destroyBootstrap(ctx context.Context, config *rest.Config, directory string
 		return errors.Wrap(err, "waiting for Kubernetes API")
 	}
 
-	events := client.CoreV1().Events("kube-system")
-
 	eventTimeout := 30 * time.Minute
 	logrus.Infof("Waiting up to %v for the bootstrap-complete event...", eventTimeout)
-	if err := waitForEvent(ctx, events, "bootstrap-complete", eventTimeout); err != nil {
+	if err := waitForEvent(ctx, client.CoreV1().RESTClient(), "event", "kube-system", "bootstrap-complete", eventTimeout); err != nil {
```

Comment:
we already know we are waiting for events, what pass `"event"` resource ?

Reply:
also are we going to wait for events in namespaces other than `kube-system`. if not overkill for now.

Reply:
> we already know we are waiting for events, what pass `"event"` resource ?

> also are we going to wait for events in namespaces other than `kube-system`. if not overkill for now.

Fixed both of these in de62f647a -> d3a4d8b31.